Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -350,6 +350,16 @@ class FakespotViewController:
              view.updateLayoutForInProgress()
              return view
 
+        case .productAdCard:
```

Comment:
This part can also be done in the integration to avoid the error when the viewModel will be removed.